src/SDKs/ServiceBus/Management.ServiceBus/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure ServiceBus management functions for managing the Microsoft Azure ServiceBus service.")]
 
 [assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
For major version bumps, `AssemblyVersion` must also be updated

Reply:
resolved